scalars/guide.md

```diff
@@ -120,40 +132,40 @@ language literal.
 
 It is used when a literal representation of a raw input value is required. For
 example the default value for a field in an Introspection response. If a GraphQL
-implementation doesn’t provide a way to define a raw input value
+implementation doesn't provide a way to define a raw input value
 programmatically, `rawInputValueToLiteral` might not be needed.
 
 ### Implementation rules
 
 These three functions need to be compatible to each other, specifically:
 
 **Rule 1:** every valid input for `parseRawInputValue` must be a valid input for
-`rawInputVariableToLiteral` and vice versa. This rule ensure that every raw
+`rawInputVariableToLiteral` and vice versa. This rule ensures that every raw
 input value can be represented as literal.
 
 **Rule 2:** for every valid input for `parseLiteral` there must at least one
 corresponding input for `parseRawInputValue` resulting in the same coerced input
-value. This rule ensure that all literals can be represented as raw input value.
+value. This rule ensures that all literals can be represented as raw input
+value.
 
 **Rule 3:** every output for `rawInputVariableToLiteral` must be a valid input
-for `parseLiteral`
+for `parseLiteral`.
 
 These rules ensure that for every valid literal we have at least one valid raw
 value and vice versa.
 
-### Coercion guidelines
+## Coercion guidelines
 
 While the specifics of the all coercion methods are implementation specific, as
-a general they should only coerce a value when no information is lost and raise
-an error otherwise.
+a general rule they should only coerce a value when no information is lost and
+raise an error otherwise.
 
 Additionally, the input coercion should be liberal in what it accepts, while the
-result coercion should be much more restricted and never different JSON values
-for the same logically value.
-
-For example a `MyLocalDate` scalar could accept the literals `"01-10-2022"` and
-`"01102022"` as input for the first of October 2022, but the result coercion
-should always return one of the possible representations.
+result coercion should be much more restricted and never produce different JSON
+values for logically identical values. For example a `MyLocalDate` scalar could
+accept the literals `"01-10-2022"` and `"01102022"` as input for the first of
+October 2022, but the result coercion should always return one of the possible
+representations.
 
 ## Scalars specification outline
```

Comment:
We should probably remove this completely and make this part of the template for specs.